addons/html_builder/static/src/core/customize_tab_plugin.js

```diff
@@ -5,6 +5,11 @@ import { registry } from "@web/core/registry";
 export class CustomizeTabPlugin extends Plugin {
     static id = "customizeTab";
     static shared = ["getCustomizeComponent", "openCustomizeComponent", "closeCustomizeComponent"];
+    resources = {
+        post_redo_handlers: () => this.closeCustomizeComponent(),
+        post_undo_handlers: () => this.closeCustomizeComponent(),
+        selectionchange_handlers: () => this.closeCustomizeComponent(),
```

Comment:
replace selectionchange_handlers by change_current_options_containers_listeners

If the selection is in the image, it will trigger a selectionchange_handlers at each preview. So you will close it to many time.

We want to close it when the selected container change i think ?

Reply:
No, we want to close it even when the selection does not change - it was the case in master. That was the bug reported by LEBL. I.e.: click outside of shape selection closes shape selection